android/src/main/java/com/swmansion/reanimated/layoutReanimation/Snapshot.java

```diff
@@ -137,4 +163,37 @@ public HashMap<String, Object> toBasicMap() {
     addBasicConfig(data);
     return data;
   }
+
+  private View findTransformedView(View view) {
+    View transformedView = null;
+    boolean isTransformed = false;
+    do {
+      if (transformedView == null) {
+        transformedView = view;
+      } else {
+        if (!(transformedView.getParent() instanceof View)) {
+          break;
+        }
+        transformedView = (View) transformedView.getParent();
+      }
+      if (transformedView == null) {
+        break;
+      }
+      float[] transformArray = new float[9];
+      transformedView.getMatrix().getValues(transformArray);
+      isTransformed =
+          transformArray[0] != 1
+              || transformArray[1] != 0
+              || transformArray[2] != 0
+              || transformArray[3] != 0
+              || transformArray[4] != 1
+              || transformArray[5] != 0
+              || transformArray[6] != 0
+              || transformArray[7] != 0
+              || transformArray[8] != 1;
```

Comment:
You could store an identity matrix somewhere (as a private static final in this class, for example), and then use `Arrays.equals` to compare them - I think that'd be a bit more clear.

Reply:
It would work if transformArray would be Float[] instead of float[]. We can use conversion like with `transformMatrix` but this requires the usage of another one loop, and I am not sure if it will be more readable.

Reply:
No, `Arrays.equals` has overloads for arrays of primitive types:
https://docs.oracle.com/javase/7/docs/api/java/util/Arrays.html#equals(float[],%20float[])

Reply:
Unless you meant something different?